tsconfig.json

```diff
@@ -2,10 +2,19 @@
   "compilerOptions": {
     "jsx": "react",
     "target": "es5",
+    "module": "es2020",
     "esModuleInterop": true,
+    "moduleResolution": "node",
     "allowSyntheticDefaultImports": true,
     "skipLibCheck": true,
-    "typeRoots": ["node_modules/@types", "node_modules/jest-extended", "types"]
+    "typeRoots": ["./node_modules/@types", "./types"]
```

Comment:
does this change matter?

Reply:
It won't make a big difference; just doing it for a bit more clarity 🌝